client/src/Components/CarDetails/inedx.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Rating from '@mui/material/Rating';
+import Typography from '@mui/material/Typography';
+import './Style.css';
+import PropTypes from 'prop-types';
+import seat from '../../images/Vector.png';
+import door from '../../images/door.png';
+import gear from '../../images/gear.png';
+
+function CarDetails({
+  name, rate, gearbox, type, doorNumber, seats, body,
```

Comment:
```suggestion
  name, rate, gearbox, type, doorNumber, seats, description,
```
I prefer to call it description, because I didn't get the meaning of it, until I complete reading the code.

Reply:
done